<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractiveLightingDetector</class>
 <widget class="QMainWindow" name="InteractiveLightingDetector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
color: black;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 95px;
max-width: 95px;
min-height: 14px;
max-height: 14px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btm_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>107</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_image">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>10</y>
      <width>441</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width : 0.5px;
border-style:solid;
border-color : black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btm_restart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>107</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>115</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>67</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>1. Segmentation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btm_StartSeg">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>107</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Start a Conture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btm_EndSingleSeg">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>107</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Done with Conture</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>2. Visualisation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btm_ShowSeg">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>107</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Segmenation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btm_ShowLV">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>107</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Lightvectors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btm_ShowAreas">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>107</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Areas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btm_EndSeg">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>107</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ready!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btm_ChooseSeg">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>107</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Conture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btm_DeleteConture">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>107</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Conture</string>
    </property>
   </widget>
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btm_backToSeg">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>300</y>
       <width>107</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Back to Segmentation</string>
     </property>
    </widget>
   </widget>
   <zorder>mainFrame</zorder>
   <zorder>btm_image</zorder>
   <zorder>lbl_image</zorder>
   <zorder>btm_restart</zorder>
   <zorder>label</zorder>
   <zorder>btm_StartSeg</zorder>
   <zorder>btm_EndSingleSeg</zorder>
   <zorder>label_2</zorder>
   <zorder>btm_ShowSeg</zorder>
   <zorder>btm_ShowLV</zorder>
   <zorder>btm_ShowAreas</zorder>
   <zorder>btm_EndSeg</zorder>
   <zorder>btm_ChooseSeg</zorder>
   <zorder>btm_DeleteConture</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
